frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -64,20 +65,90 @@ local function tidy_markup(results)
     return results
 end
 
+function ReaderDictionary:cleanSelection(text, is_single_word)
```

Comment:
I finally got rid of is_single_word. It was there because in readerdictionary, there was a call to stripePunctuation, that wasn't there in readerwikipedia. So i thought i would do as it was done, but stripePunctuation is as well useful in both cases.